app/uk/gov/hmrc/helptosavereminder/controllers/HtsUserUpdateController.scala

```diff
@@ -77,15 +75,15 @@ class HtsUserUpdateController @Inject()(
     } else notAllowedThisNino
   }
 
-  def deleteHtsUser(): Action[AnyContent] = ggAuthorisedWithNino { implicit request => implicit nino ⇒
+  def deleteHtsUser(): Action[AnyContent] = Action.async { implicit request =>
```

Comment:
This didn't use a nino so I didn't think the ggAuthorisedWithNino was necessary.